<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <class>PredefinedPset</class>
    <widget class="QDialog" name="PredefinedPset">
        <property name="geometry">
            <rect>
                <x>0</x>
                <y>0</y>
                <width>659</width>
                <height>492</height>
            </rect>
        </property>
        <property name="contextMenuPolicy">
            <enum>Qt::ContextMenuPolicy::CustomContextMenu</enum>
        </property>
        <property name="windowTitle">
            <string>Dialog</string>
        </property>
        <layout class="QGridLayout" name="gridLayout">
            <item row="2" column="0" colspan="2">
                <widget class="QDialogButtonBox" name="buttonBox">
                    <property name="orientation">
                        <enum>Qt::Orientation::Horizontal</enum>
                    </property>
                    <property name="standardButtons">
                        <set>QDialogButtonBox::StandardButton::Ok</set>
                    </property>
                </widget>
            </item>
            <item row="1" column="0">
                <widget class="QListWidget" name="list_view_pset">
                    <property name="focusPolicy">
                        <enum>Qt::FocusPolicy::StrongFocus</enum>
                    </property>
                    <property name="contextMenuPolicy">
                        <enum>Qt::ContextMenuPolicy::CustomContextMenu</enum>
                    </property>
                    <property name="editTriggers">
                        <set>QAbstractItemView::EditTrigger::EditKeyPressed</set>
                    </property>
                    <property name="sortingEnabled">
                        <bool>false</bool>
                    </property>
                </widget>
            </item>
            <item row="0" column="0">
                <widget class="QLabel" name="label">
                    <property name="text">
                        <string>PropertySet</string>
                    </property>
                </widget>
            </item>
            <item row="0" column="1">
                <widget class="QLabel" name="label_2">
                    <property name="text">
                        <string>Inherits to:</string>
                    </property>
                </widget>
            </item>
            <item row="1" column="1">
                <widget class="QTableWidget" name="table_widgets_classes">
                    <property name="contextMenuPolicy">
                        <enum>Qt::ContextMenuPolicy::CustomContextMenu</enum>
                    </property>
                    <property name="editTriggers">
                        <set>QAbstractItemView::EditTrigger::NoEditTriggers</set>
                    </property>
                    <property name="selectionMode">
                        <enum>QAbstractItemView::SelectionMode::ExtendedSelection</enum>
                    </property>
                    <property name="sortingEnabled">
                        <bool>true</bool>
                    </property>
                    <attribute name="horizontalHeaderStretchLastSection">
                        <bool>true</bool>
                    </attribute>
                    <attribute name="verticalHeaderVisible">
                        <bool>false</bool>
                    </attribute>
                    <column>
                        <property name="text">
                            <string>Name</string>
                        </property>
                    </column>
                    <column>
                        <property name="text">
                            <string>Identifier</string>
                        </property>
                    </column>
                </widget>
            </item>
        </layout>
    </widget>
    <resources />
    <connections>
        <connection>
            <sender>buttonBox</sender>
            <signal>accepted()</signal>
            <receiver>PredefinedPset</receiver>
            <slot>accept()</slot>
            <hints>
                <hint type="sourcelabel">
                    <x>248</x>
                    <y>254</y>
                </hint>
                <hint type="destinationlabel">
                    <x>157</x>
                    <y>274</y>
                </hint>
            </hints>
        </connection>
        <connection>
            <sender>buttonBox</sender>
            <signal>rejected()</signal>
            <receiver>PredefinedPset</receiver>
            <slot>reject()</slot>
            <hints>
                <hint type="sourcelabel">
                    <x>316</x>
                    <y>260</y>
                </hint>
                <hint type="destinationlabel">
                    <x>286</x>
                    <y>274</y>
                </hint>
            </hints>
        </connection>
    </connections>
</ui>